instructions for requesting new packages (#655)
coreos/fedora-coreos-tracker: NEWPACKAGE.md

# Request to Include a New Package in Fedora CoreOS

If you would like to propose the inclusion of a new package into the base
content set of Fedora CoreOS, please open a [new issue](https://github.com/coreos/fedora-coreos-tracker/issues/new) 
with the following questions answered.  The more detail provided for each
question, the better informed everyone will be.

Please title the new issue: `Package Request: <name of package>`

> 1. What, if any, are the additional dependencies on the package? (i.e. does it drag in Python, Perl, etc)

Reviewer: Nit: nicer phrasing:

```suggestion
1. What, if any, are the additional dependencies on the package? (i.e. does it pull in Python, Perl, etc)
```

2. What is the size of the package and its dependencies?
3. What problem are you trying to solve with this package? Or what functionality does the package provide?

> 4. Can the package be run from a container?  Explain why or why not.

Reviewer 2: ```suggestion
4. Can the software provided by the package be run from a container?  Explain why or why not.
```

> 5. Can the tool be helpful in debugging container runtime issues?

Reviewer 2: ```suggestion
5. Can the tool(s) provided by the package be helpful in debugging container runtime issues?
```

> 6. Can the tool be helpful in debugging networking issues?

Reviewer 2: ```suggestion
6. Can the tool(s) provided by the package be helpful in debugging networking issues?
```

7. Is it possible to layer the package onto the base OS as a day 2 operation?  Explain why or why not.

> 8. Can the packaging be adjusted to just deliver binaries?

Reviewer 2: ```suggestion
8. In the case of packages providing services and binaries, can the packaging be adjusted to just deliver binaries?
```

> 9. Can the tool be used to do things we’d rather users not be able to do in FCOS? (e.g. can it be abused as a Turing complete interpreter?)

Reviewer 2: ```suggestion
9. Can the tool(s) provided by the package be used to do things we’d rather users not be able to do in FCOS? (e.g. can it be abused as a Turing complete interpreter?)
```

> 10. Does the tool have a history of CVEs?

Reviewer 2: ```suggestion
10. Do the software provided by the package have a history of CVEs?
```

Reviewer: Should be "Does the software...", no?

# Request to Include a New Package in Fedora CoreOS

If you would like to propose the inclusion of a new package into the base
content set of Fedora CoreOS, please open a [new issue](https://github.com/coreos/fedora-coreos-tracker/issues/new) 

> here on the [fedora-coreos-tracker repo](https://github.com/coreos/fedora-coreos-tracker) 

Reviewer 3: Hmm, given that we link to the new issue, probably OK to just drop this entire line.

Author: Yeah, not sure why I thought to be overly verbose
